frontend/src/view/audio-annotator/audio-annotator.page.tsx

```diff
@@ -200,52 +209,59 @@ export const AudioAnnotator: React.FC = () => {
       {/* Header */ }
       <div className="row">
         <h1 className="col-sm-6">APLOSE</h1>
-        <p className="col-sm-4 annotator-nav">
-              <span>
-                <a href="https://github.com/Project-OSmOSE/osmose-app/blob/master/docs/user_guide_annotator.md"
-                   rel="noopener noreferrer"
-                   target="_blank">
-                  <span className="fas fa-question-circle"></span>&nbsp;Annotator User Guide
-                </a>
-              </span>
-          { context.instructionsURL &&
-              <span>
-                      <a href={ context.instructionsURL }
-                         rel="noopener noreferrer"
-                         target="_blank">
-                          <span className="fas fa-info-circle"></span>&nbsp;Campaign instructions
-                      </a>
-                  </span> }
-        </p>
+        <div className="col-sm-4 annotator-nav align-items-center gap-1">
+          <IonButton color="secondary"
+                     fill={ "outline" }
+                     onClick={ openGuide }>
+            <IonIcon icon={ helpCircle } slot="start"/>
+            Annotator user guide
+          </IonButton>
+
+          { instructionsURL && <IonButton color="secondary"
+                                          fill={ "outline" }
+                                          onClick={ openInstructions }>
+              <IonIcon icon={ informationCircle } slot="start"/>
+              Campaign instructions
+          </IonButton> }
+        </div>
         <ul className="col-sm-2 annotator-nav">
           <li>
-            <Link className="btn btn-danger"
-                  to={ `/annotation_tasks/${ context.campaignId }` }
-                  title="Go back to annotation campaign tasks">
+            <IonButton color="danger"
+                       onClick={ goBack }
+                       title="Go back to annotation campaign tasks">
               Back to campaign
-            </Link>
+            </IonButton>
           </li>
         </ul>
       </div>
 
       {/* Audio player (hidden) */ }
-      <AudioPlayerComponent ref={ audioPlayerRef }/>
+      <AudioPlayerComponent ref={ ref => {
+        audioPlayerRef.current = ref;
+        console.debug()
```

Comment:
Is there a reason to keep this `console.debug()`?

Reply:
It has been removed, sorry for that